react/components/atoms/button-tab-text/button-tab-text.js

```diff
@@ -93,6 +95,7 @@ export class ButtonTabText extends mix(PureComponent).with(IdentifiableMixin) {
                 <Touchable
                     style={this._touchableStyle()}
                     disabled={this.props.disabled}
+                    underlayColor={this.props.underlayColor}

```

Comment:
Unfortunately this will require extra changes, since the underlay causes different styling and is not adapting to the border radius 😕

https://user-images.githubusercontent.com/25725586/160136320-cd56f8c0-2cfb-4842-a8fc-92ce18f31ac1.mov

Reply:
If you want I can see this while your iOS setup has problems

Reply:
Can you please just check it again? For the overflow problem on my end it looks like it's fixed.

Reply:
It does not happen in Android, only in iOS since this underlay color is only used for ios 😞 
THis is nitpicking at this point, but here is up close:

https://user-images.githubusercontent.com/25725586/160175211-43a82c0a-bed4-42d4-883e-64d9b8399afb.mov

Reply:
It's weird that https://github.com/ripe-tech/ripe-components-react-native/pull/312/commits/3bed9c8d58979cd8573f000f149f51cdb67ef11b didn't fix the overflow of the borders. I'm won't be able to check this sooner on iOS since my setup is temporarily broken on compilation errors for iOS as you know so if you have the time and it is simple enough, I'd appreciate it! :pray: